data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -141,7 +141,8 @@ then
 		--asset-input-dir=/assets/tls \
 		--asset-output-dir=/assets/kube-apiserver-bootstrap \
 		--config-output-file=/assets/kube-apiserver-bootstrap/config \
-		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml
+		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml \
+		--cluster-auth-file=/assets/manifests/cluster-authentication-02-config.yaml
```

Comment:
trying to work out where this comes from.  Is this filename `cluster-authentication-02-config.yaml` actually produced by the cluster-config-operator?